docs/topic-guides/hp-constraints-det.txt

```diff
@@ -0,0 +1,85 @@
+.. _topic-guides_hp-constraints-det:
+
+###################################
+ Hyperparameter Search Constraints
+###################################
+
+Determined's Hyperparameter Search Constraints API enables users to
+specify their hyperparameter searches in a more granular way beyond the
```

Comment:
Determined's Hyperparameter Search Constraints API enables finer grained control over the hyperparameter search space through the specification of additional constraints.  Among other things, this functionality is particularly useful for incorporating prior knowledge/domain expertise into a hyperparameter search and constraining the search to models that fit a particular deployment environment.